### DISCUSSION TITLE
what does (0) mean in finder info?

### DISCUSSION THREAD

**Discussion by tg0h** [junegunn/fzf#3058] [Q&A] [answered]
<img width="489" alt="image" src="https://user-images.githubusercontent.com/7954426/203318678-f0ba7221-e1b4-4361-a90d-c6eb7fad8c02.png">

I'm not sure if this was just introduced in fzf recently or if I've missed this all the time I've been using fzf... what does  the 0 mean in the finder info?

**Comment by Delapouite** [comment, marked as answer]
If you enable the `--multi` flag, you're able to select several item in the list with the TAB key. So this 0 increases each time you toggle a candidate.
